### DISCUSSION TITLE
How does the minio cluster determine the size of the erasure set?

### DISCUSSION THREAD

**Discussion by zhaowq32** [minio/minio#18668] [Q&A]
## NOTE
The impact of Erasure Code Stripe Size and Erasure Code Parity on capacity and fault tolerance can be calculated through the erasure code calculator.The erasure code calculator can choose different Erasure Code Stripe Sizes. Can this value be set in the minio cluster?If not, what is the calculation logic of Erasure Code Stripe Size?Can I get this value through the mc command?

**Comment by kietheros** [comment]
You need to config env MINIO_STORAGE_CLASS_STANDARD before running minio server.
Ex: MINIO_STORAGE_CLASS_STANDARD=EC:2
You can check the default config of minio cluster at here `https://github.com/minio/minio/tree/master/docs/erasure/storage-class`

**Reply by zhaowq32** [reply]
MINIO_STORAGE_CLASS_STANDARD and MINIO_STORAGE_CLASS_RRS are configurations for setting Erasure Code Parity(M), not Erasure Code Stripe Size(K+M).

**Reply by kietheros** [reply]
K+M = N is the number of disks in cluster.

**Reply by zhaowq32** [reply]
If the value of K+M is the number of disks in the cluster, then what is the point of modifying the value of Erasure Code Stripe Size in the erasure calculator?[https://min.io/product/erasure-code-calculator](url).In addition, the size of the erasure set is 2-16.

**Reply by kietheros** [reply]
@zhaowq32 Sorry, I just tried one server and many disks. I also don't understand Stripe Size very well.

**Reply by zhaowq32** [reply]
Still thanks for your reply

**Reply by kietheros** [reply]
@zhaowq32 Have you found a way to set erasure set size ?

**Comment by Jazz9** [comment]


**Reply by klauspost** [reply]
@Jazz9 Please stop hijacking other peoples questions with your own. We have told you multiple times to get a subscription if you want this type of help.

**Reply by Jazz9** [reply]
I wasn't asking for official support - I was asking fellow minio users if they knew how to set the K value. ( and you have since told me in the other thread that it is automatic - thanks! )
